_pages/how-we-work/tools/anyconnect.md

```diff
@@ -2,46 +2,10 @@
 title: AnyConnect
 ---
 
-AnyConnect is a VPN client that we use to connect to GSA's intranet.
-
-## Setup
-
-### VPN Client
-Your machine may already have AnyConnect installed. If it is, it will be under Cisco>AnyConnect in your Applications.
-
-If it is not installed, here is how you can get access:
-
-1. Request the AnyConnect VPN Client via [ServiceNow](https://gsa.service-now.com/sp?id=sc_cat_item&sys_id=1bfdfdca78d3a400ce3ddff91a64940b&sysparm_category=b628125b7cec0100a6e757fe35f45fb8). Someone from GSA IT will reach out to you within the day.
-2. Follow their directions to install AnyConnect.
-
-Your installation screen should look like this:
-
-![AnyConnect Installation Screen]({{site.baseurl}}/images/anyconnect/1.png)
-
-## Logging into Cisco AnyConnect
-
-  1. Click on Launchpad (Rocket ship icon in the Dock at the bottom of your screen) 
-  2. Click on the Cisco AnyConnect Mobility Client icon to launch
-  3. Select your server in the pop-up window and click connect
-  4. Enter ENT login and password
-  5. Go to [otp.gsa.gov](http://otp.gsa.gov) and enter your ENT info to get the Tokencode
-     - You can also obtain a Tokencode by installing [SecureAuth OTP](https://preview-insite.gsa.gov/employee-resources/information-technology/do-it-yourself-self-help/telework-technology/secureauth/install-secureauth-otp-on-ios-apple)) on your phone, as described in the [Get a one-time password?]({{site.baseurl}}/distributed/#get-a-one-time-password) handbook section
-  6. Enter the Tokencode back into thee AnyConnect pop-up screen and submit
-
-## Usage
-
-Refer to [Insite](https://preview-insite.gsa.gov/employee-resources/information-technology/do-it-yourself-self-help/telework-technology/virtual-private-network-vpn/vpn-to-mac-os-quick-setup#connect) for the latest instructions on how to use the AnyConnect VPN Client. 
-
-You will need to enter your GSA credentials to access this page. 
-
-## Troubleshooting
-
-If you need to troubleshoot your AnyConnect installation, please refer to [this document](https://docs.google.com/document/d/1dighScy4bN346VYXfbZpkBu2TDZaduaurA_lOuXBYfo/)
+Cisco AnyConnect is the VPN client that we use to connect to GSA's network. See [how to connect using your PIV card](https://gsa.service-now.com/sp/?id=kb_article&sys_id=6d6753d1dbb73b84c3d674608c96195c), or if you're having PIV issues, [using SecureAuth](https://gsa.service-now.com/sp/?id=kb_article&sys_id=22191902db10774058c2fd721f9619f5).
```

Comment:
A few thoughts:
1) I use anyconnect, but so far have not seen how it relates to PIV readers, so I'm not getting the connection here and I'm not sure what I'm missing.
2) As an anyconnect user, my most common trouble point is that it loses the `vpn.gsa.gov/gfeotp` connection string, and that's what I go to the handbook looking for. Even with the existing instructions, it was annoying that the piece of information I most needed was hidden behind multiple logins. If there was one piece of info to retain in the handbook, that would be my vote.
3. Pulling out the intial setup and install info makes a lot of sense to me. I'm less sure about the "logging in" info.

Reply:
> I use anyconnect, but so far have not seen how it relates to PIV readers, so I'm not getting the connection here and I'm not sure what I'm missing.

Did you take a look at the newly linked documentation?

> my most common trouble point is that it loses the `vpn.gsa.gov/gfeotp` connection string

How recently has that happened? I don't believe that connection string should be required anymore.

No to belabor the point, but this is another example where our Handbook was providing out-of-date information, while the Knowledge Base articles exist and are clear and accurate.

Reply:
> I use anyconnect, but so far have not seen how it relates to PIV readers, so I'm not getting the connection here and I'm not sure what I'm missing.

Did you take a look at the newly linked documentation?

> my most common trouble point is that it loses the `vpn.gsa.gov/gfeotp` connection string

How recently has that happened? I don't believe that connection string should be required anymore.

No to belabor the point, but this is another example where our Handbook was providing out-of-date information, while the Knowledge Base articles exist and are clear and accurate.

Reply:
OK, I've looked at the newly linked docs now. Those must be pretty new, right? I followed the link to the older insite doc as recently as a week or so ago -- to get that connection string again :-)

Reply:
> Those must be pretty new, right?

🤷‍♂